docs/docs/external/getting-started/test-network/setup-packages.md

```diff
@@ -0,0 +1,346 @@
+---
+id: setup-packages
+title: Setup with Imported Weaver Components
+---
+
+<!--
+ Copyright IBM Corp. All Rights Reserved.
+
+ SPDX-License-Identifier: CC-BY-4.0
+ -->
+
+In this document, we detail the steps using which you can bring up networks using the default configuration settings and by fetching pre-built Weaver interoperation modules, SDK libraries, and relay drivers from Github Package repositories. To customize these settings (e.g., hostnames, ports), refer to the [Advanced Configuration page](./advanced-configuration.md).
+
+_Note_: The default configuration is for a development setup, therefore all components are run on `localhost`, many within Docker containers.
+
+Follow the instructions below to build and run components followed by interoperation flows. These instructions have been tested on Ubuntu Linux (bash shell) and Mac OS. In general, they should work on any system and shell as long as the various dependenices have been installed and configured.
+
+## Prerequisites
+
+### Software
+Before starting, make sure you have the following software installed on your host machine:
+- Curl: _install using package manager, like `apt` on Debian/Ubuntu Linux_
+- Git: [sample instructions](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+- Docker: [sample instructions](https://docs.docker.com/engine/install/) (Latest version)
+- Docker-Compose: [sample instructions](https://docs.docker.com/compose/install/) (Latest version)
+- Golang: [sample instructions](https://golang.org/dl/) (Version 1.15 or above)
+- Java (JDK and JRE): [sample instructions](https://openjdk.java.net/install/) (Version 8)
+- Node.js and NPM: [sample instructions](https://nodejs.org/en/download/package-manager/) (Version 11 to Version 14 Supported)
+- Yarn: [sample instructions](https://classic.yarnpkg.com/en/docs/install/)
+- Rust: [sample instructions](https://www.rust-lang.org/tools/install)
+  * To avoid errors during Weaver Relay compilation, update certain packages (on which the Weaver Relay is depenent) to their latest versions as follows:
+    ```
+    cargo update -p nom
+    cargo update -p lexical-core
+    ```
+- Protoc (Protobuf compiler): _Golang should already be installed and configured._
```

Comment:
Protoc is not required when using github packages.  (as we will fetch built protos from github).

Reply:
Fixed